src/utils/local/wrap-display-name.js

```diff
@@ -0,0 +1,21 @@
+// Returns a new display name for a Component combining the Base Component name with the name of the HOC.
+
+function wrapDisplayName(BaseComponent, hocName) {
+  return `${hocName}(${getDisplayName(BaseComponent)})`
+}
+
+// Helper
+
+function getDisplayName(Component) {
+  if (typeof Component === 'string') {
+    return Component
+  }
+
+  if (!Component) {
+    return undefined
+  }
+
+  return Component.displayName || Component.name || 'Component'
+}
```

Comment:
I didn't pull this out separately since we weren't using it separately originally.